packages/Select/tests/Select.spec.js

```diff
@@ -22,4 +22,34 @@ describe("Select", () => {
     );
     expect(getByText(/placeholder value/i)).toBeInTheDocument();
   });
+
+  it("should not display the options", () => {
+    const { queryByText } = render(
+      <Select placeholder="placeholder value">
+        <option value="Select value">Select value</option>
+        <option value="Another option">Another option</option>
+      </Select>
+    );
+    expect(queryByText(/place holder value/i)).not.toBeInTheDocument();
```

Comment:
I think it's better to test if "Select value" and "Another option" are not in the DOM instead of place holder value for this scenario.